playbook/app/pb_kits/playbook/pb_icon_circle/_icon_circle.tsx

```diff
@@ -26,23 +27,41 @@ type IconCircleProps = {
 }
 
 const IconCircle = (props: IconCircleProps) => {
-  const { aria = {}, className, dark = false, data = {}, icon, id, size = 'md', variant = 'default' } = props
+  const { aria = {}, className, dark = false, data = {}, emoji, icon, id, size = 'md', variant = 'default' } = props
 
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const classes = classnames(buildCss('pb_icon_circle_kit', size, variant), globalProps(props), className)
 
+  const isValidEmoji = (emoji: string) => {
+    // Using regular expression to check if the string is a valid emoji/emoji Unicode
+    const emojiRegex = /^(\p{Emoji}|\uFE0F)+$/u;
+    return emojiRegex.test(emoji);
+  };
+
```

Comment:
Added validation for emojis. The emoji prop takes a string so added this validation to make sure that a random string being passed in would not just render that string to the page. Tested this and if anything is passed to the prop that is not a valid emoji or emoji unicode, nothing renders to the circle. This is the same as for the icon prop, if anything is passed to icon prop that is not a valid icon, it renders nothing to the circle.